arlo_server/routes.py

```diff
@@ -878,20 +882,37 @@ def ballot_set(election_id, jurisdiction_id, batch_id, ballot_position):
             ),
             404,
         )
-    elif len(ballots) > 1:
+
+    # explicitly remove existing interpretations in case this ballot was previously set.
+    ballot.interpretations = []
+    ballot.status = BallotStatus.NOT_FOUND
```

Comment:
I was imagining doing this within the same endpoint so we don't have to keep things consistent between diff endpoints. E.g. the ballot set endpoint would take a body with this type:

```
{
  status: BallotStatus
  interpretations: List[Interpretation]
}
```

Any reason you opted for a separate endpoint?

Reply:
In this case I chose a separate endpoint because I am biased towards having frontend <--> backend APIs be the ones that can't change as often (as internal frontend ones like the other comment), and so wanted these two cases to be very different. Could do it your way but then that would change a lot of existing behavior and that felt riskier with not a ton of benefit (I don't see many other statuses being added.)